Formula/j/julia.rb

```diff
@@ -91,25 +91,23 @@ def install
     args << "MACOSX_VERSION_MIN=#{MacOS.version}" if OS.mac?
 
     # Set MARCH and JULIA_CPU_TARGET to ensure Julia works on machines we distribute to.
-    # Values adapted from https://github.com/JuliaCI/julia-buildkite/blob/main/utilities/build_envs.sh,
-    # then extended.
+    # Values adapted from https://github.com/JuliaCI/julia-buildkite/blob/main/utilities/build_envs.sh
     args << "MARCH=#{Hardware.oldest_cpu}" if Hardware::CPU.intel?
 
     cpu_targets = %w[generic]
     if Hardware::CPU.arm?
-      # Cortex A57, Thunder-X2, Nvidia Carmel, Neoverse V1/V2,
-      # Cortex A510/A710/A715/X2/X3
-      if OS.linux?
-        cpu_targets += %w[cortex-a57 thunderx2t99 carmel,clone_all neoverse-512tvb,base(3)
-                          armv8.5-a,sve2,sve2-bitperm,i8mm,bf16,base(3)]
+      if OS.mac?
+        # For Apple Silicon, we don't care about other hardware
+        cpu_targets << "apple-m1,clone_all"
```

Comment:
```suggestion
        cpu_targets << "apple-m1,base(3)"
```
https://github.com/JuliaCI/julia-buildkite/blob/ea50eb719242bc3e844227a2ebf54a49d308bafc/utilities/build_envs.sh#L67

Reply:
`base(3)` means a profile is built on top of another base, and we don't want that here. We really want `apple-m1,clone_all`. Confirmed by upstream at https://github.com/JuliaLang/julia/issues/54319#issuecomment-2087587703